builtin/mainmenu/pkgmgr.lua

```diff
@@ -114,6 +133,12 @@ function pkgmgr.get_folder_type(path)
 		return { type = "mod", path = path }
 	end
 
+	testfile = io.open(path .. DIR_DELIM .. "modpack.conf","r")
```

Comment:
why not add a helper function for file_exists ? this is called in many placed and is error prone. an helper can help :)

Reply:
out of scope for this PR, really